src/js/components/Meter/stories/Bar.js

```diff
@@ -4,12 +4,14 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, Meter } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const BarMeter = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      <Meter type="bar" background="light-2" values={[{ value: 30 }]} />
-    </Box>
-  </Grommet>
-);
+function BarMeter() {
```

Comment:
There is a reason to change from `const` to `function`?
Also, you could create a variable to store `[{ value: 30 }]` and avoid unnecessary re-render:
```js
const meterValues = [{ value: 30 }];

const BarMeter = () => (
  <Grommet theme={grommet}>
    <Box align="center" pad="large">
      <Meter type="bar" background="light-2" values={meterValues} />
    </Box>
  </Grommet>
);
```

Reply:
I see now that other refactored code is using lambdas so I will do that for consistency, though personally I normally only use those if it's getting passed as an event handler to avoid the need for .bind(this)

I'm curious about the values though. It's never being changed in the parent component, so whether I pass it as a literal or a variable, it wouldn't effect how often it is re-rendered?

I did create a variable in labelled.js as it was being used in more than one location, but again for consistency I will fix this :)